runtime/runtime.go

```diff
@@ -786,7 +802,18 @@ func validateArgumentParams(
 			}
 		}
 
-		arg, err := importValue(inter, value, parameterType)
+		var arg interpreter.Value
+		maybeErr := panicToInvalidPublicKeyError(func() {
```

Comment:
Needed to wrap the panic caused by validation of invalid public key, which happens in `importValue`.